com.github.hopsan.Hopsan.json

```diff
@@ -17,25 +17,14 @@
         "--filesystem=home"
     ],
     "modules": [
-        {
-            "name": "doxygen",
-            "buildsystem": "cmake",
-            "config-opts": [ "-DCMAKE_BUILD_TYPE=Release" ],
-            "sources": [
-                {
-                    "type": "archive",
-                    "url": "https://github.com/doxygen/doxygen/archive/Release_1_8_20.tar.gz",
-                    "sha256": "3dbdf8814d6e68233d5149239cb1f0b40b4e7b32eef2fd53de8828fedd7aca15"
-                }
-            ],
-            "cleanup": [ "*" ]
-        },
         {
             "name": "hopsan",
             "disabled": false,
             "buildsystem": "simple",
             "build-commands": [
-                "./packaging/prepareSourceCode.sh $(pwd) $(pwd) 2.19.0 20210510.1936 2.19.0.20210510.1936 false false",
+                "pwd",
+                "ls -l",
+                "./packaging/prepareSourceCode.sh $(pwd) $(pwd) 2.19.1 20211105.2150 2.19.1.20211105.2150 false false",
 
                 "cd dependencies; ./setupDiscount.sh",
```

Comment:
the current directory isn't reset at each comand,

Reply:
Aha, thank you for telling me. This worked the last time I released about 6 month ago.